tools/gen_ort_dockerfile.py

```diff
@@ -389,58 +459,13 @@ def dockerfile_for_windows(output_file):
                         help='Home directory for TensorRT.')
 
     FLAGS = parser.parse_args()
+    preprocess_gpu_flags()
```

Comment:
conditionalize `preprocess_gpu_flags` call within `if FLAGS.enable_gpu:` and remove the flag from within the function.